### PR TITLE
New version: Ryujinx.Ryujinx.Ava version 1.1.1217

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/141747)

### CODE REVIEW

File: manifests/r/Ryujinx/Ryujinx/Ava/1.1.1217/Ryujinx.Ryujinx.Ava.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created with komac v2.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Ryujinx.Ryujinx.Ava
+PackageVersion: 1.1.1217
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles: []
```
Comment (Exorcism0666) [inline manifests/r/Ryujinx/Ryujinx/Ava/1.1.1217/Ryujinx.Ryujinx.Ava.installer.yaml:8]: @russellbanks I don't know why it appears, but in the recent PRs opened on my account, "NestedInstallerFiles: []" appears.

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/r/Ryujinx/Ryujinx/Ava/1.1.1217/Ryujinx.Ryujinx.Ava.installer.yaml (hunk 1 above)
Comment (russellbanks) [reply manifests/r/Ryujinx/Ryujinx/Ava/1.1.1217/Ryujinx.Ryujinx.Ava.installer.yaml:8]: Thanks for reporting. This will be an easy fix :)

Comment (russellbanks) [reply manifests/r/Ryujinx/Ryujinx/Ava/1.1.1217/Ryujinx.Ryujinx.Ava.installer.yaml:8]: Fixed in https://github.com/russellbanks/Komac/commit/d984c065d2a986adf1a56e5657cc3e7deb6e1cf1

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-03-02T12:51:13.0204419Z ##[error] Manifest Error: Required field missing. [NestedInstallerFiles]

(Automated response - build 861.)

**Pull request closed without merging**